tests/e2e/performance/csi_tests/test_pvc_bulk_clone_performance.py

```diff
@@ -49,113 +52,150 @@ def test_bulk_clone_performance(self, namespace, tmp_path, pod_factory):
             sc_name = constants.DEFAULT_STORAGECLASS_CEPHFS
             clone_yaml = constants.CSI_CEPHFS_PVC_CLONE_YAML
 
-        pvc_dict_list = scale_lib.construct_pvc_creation_yaml_bulk_for_kube_job(
-            no_of_pvc=pvc_count,
-            access_mode=constants.ACCESS_MODE_RWO,
-            sc_name=sc_name,
-            pvc_size="5Gi",
-        )
-
-        job_pvc_file = ObjectConfFile(
-            name="job_profile_pvc",
-            obj_dict_list=pvc_dict_list,
-            project=self.namespace,
-            tmp_path=tmp_path,
-        )
-
-        # Create kube_job
-        job_pvc_file.create(namespace=self.namespace)
-
-        # Check all the PVC reached Bound state
-        pvc_bound_list = scale_lib.check_all_pvc_reached_bound_state_in_kube_job(
-            kube_job_obj=job_pvc_file,
-            namespace=self.namespace,
-            no_of_pvc=pvc_count,
-        )
-
-        logging.info(f"Number of PVCs in Bound state {len(pvc_bound_list)}")
-
-        total_files_size = self.run_fio_on_pvcs(pvc_dict_list, pod_factory)
```

Comment:
we need this line to fill up the pvc with data - or implement it in a different way but this most be done before the clone.

Reply:
Agree with Avi. First - fill pvc with data and only after create clones.

Reply:
This piece of code is there in line 107, since I added some new lines for creating pod using kube_job it's shown in red by github.